src/extension/common/components/copyButton.re

```diff
@@ -1,4 +1,10 @@
-let createElement ::label="copy" ::text ::onCopy ::style=? children::_ () =>
-  <CopyToClipboard text onCopy>
-    <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
-  </CopyToClipboard>;
+
+let component = ReasonReact.statelessComponent "CopyButton";
+let make ::label="copy" ::text ::onCopy ::style=? _ => {
+  ...component,
+
+  render: fun _ _ =>
+    <CopyToClipboard text onCopy>
+      <span className="copy-button" style=?style> (ReactRe.stringToElement label) </span>
+    </CopyToClipboard>
+};
```

Comment:
Is it necessary to turn it into a component, or can the function just be used directly?